### PR TITLE
(webhooks/php) add integrity check code

### PR DESCRIPTION
**Pull request by eteeselink**
Got nerd sniped by the corresponding issue being closed.

Tested locally with the PHP devserver, works excellently. Also included the customer-provided Laravel versions of the same functionality for completeness, although I'm pretty confident that the vanilla PHP code will work in most Laravel setups too.

### CODE REVIEW

File: webhooks/php/webhook_callback.php [hunk 1]
```diff
@@ -1,10 +1,36 @@
 <?php
-// Read more on https://talkjs.com/docs/Webhooks/Getting_Started.html
-// Parse the request body as JSON
+// This is a vanilla PHP example to read a webhook and validate its integrity.
```
Comment (bigblind) [inline webhooks/php/webhook_callback.php:2]: nitpick: I find the wording here slightly weird. I'd change it to

> This is a vanilla PHP example of how to receive webhook requests and validate the signature, so you can be certain that the request was sent by TalkJS.

File: webhooks/php/webhook_callback.php [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+//   $payload = $request->getContent();
+//   $signature = $request->header("X-TalkJS-Signature");
+//   $timestamp = $request->header("X-TalkJS-Timestamp");
+
+
+// Your secret key, as found in the TalkJS dashboard. Make sure that you never
+// share your secret key anywhere nor commit it to code repositories.
+$secret = "YOUR_SECRET_KEY";
+// Laravel:
+//   $secret = env("TALKJS_SECRET_KEY");
```
Comment (bigblind) [inline webhooks/php/webhook_callback.php:20]: Maybe make the laravel and PHP examples behave the same here, and change the vanilla example to read from an environment variable as well? Now the way the two examples behave differently feels weird to me. Then again, maybe pulling environment variables into this adds too much complexity for someone who just wants to plunk this file on a basic apache server and be done with it.

**Review by bigblind — APPROVED** [review]
LGTM. 2 minor comments, but nothing that feels merge-blocking.

File: webhooks/php/webhook_callback.php (hunk 1 above)
Comment (eteeselink) [reply webhooks/php/webhook_callback.php:2]: Great point, taking over your suggestion verbatim.

**Comment by eteeselink** [comment]
Great feedback, thanks! Gonna merge away, lemme know if you have further comments anyway.

**Pull request merged**